packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Updates minimum Flutter version to 3.0.
+* Ignores a lint in the tests for backwards compatibility.
```

Comment:
All of the changelog and version changes should be reverted; the repo tooling currently doesn't do line-by-line analysis so don't know that (non-doc) comments are exempt.

Reply:
Done.